packages/ui/feature-builder-right-sidebar/src/lib/step-type-sidebar/step-type-sidebar.component.ts

```diff
@@ -110,33 +103,20 @@ export class StepTypeSidebarComponent implements OnInit, AfterViewInit {
     this.showRequestPiece$ = this.flagsService.isFlagEnabled(
       ApFlagId.SHOW_COMMUNITY
     );
-    //EE
     this.searchControlTelemetry$ = this.searchFormControl.valueChanges.pipe(
       debounceTime(1500),
       filter((val) => !!val),
-      switchMap((val) => {
+      tap((val) => {
         this.telemetryService.capture({
           name: TelemetryEventName.PIECES_SEARCH,
           payload: {
             target: this._showTriggers ? 'triggers' : 'steps',
             search: val || '',
           },
         });
-        return this.telemetryService.savePiecesSearch({
-          target: this._showTriggers ? 'triggers' : 'steps',
-          search: val || '',
-          insideTemplates: false,
-        });
```

Comment:
the save pieces search function was meant to be saving stuff in a sheet and inside templates was meant for when users search for pieces in the templates dialog, just making sure this wasn't removed by accident.